modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf

```diff
@@ -5,6 +5,6 @@ BlueBoard removeAction GVAR(canceledBlu);
 
 [-2, {hint "Blufor has signaled they are ready to begin!";}] call CBA_fnc_globalExecute;
 
-GVAR(bluforIsReady) = True;
+SETMVAR(bluforIsReady, True);
 BlueBoard hideObjectGlobal true;
```

Comment:
doesn't hideObjectGlobal need to be executed on the server? From the server using that command to propagate to all clients?

Reply:
Done.